### DISCUSSION TITLE
anvil_orm future?

### DISCUSSION THREAD

**Discussion by hugetim** [anvilistas/anvil-labs#84] [General]
@meatballs, do you see 'historic' as a replacement for 'anvil_orm', or as something complementary? Mostly I'm wondering whether it's worth taking some time to learn 'anvil_orm' at this point and contribute to its docs.

**Comment by meatballs** [comment]
I'd say this is pretty much superseded by historic now.

**Comment by hugetim** [comment]
Ok, that makes sense. I might try to draft some historic documentation then
(as part of my learning process--no worries if it proves unworthy for
pulling into the docs for real).

**Comment by meatballs** [comment]
Also, it doesn't take into account the new accelerated tables. Perhaps, if I thought that through properly and implemented whatever changes are necessary, I could resurrect this alongside historic, but I just don't have the time.

**Reply by hugetim** [reply]
Ok, that makes sense. I'm also not likely to have time to work drafting historic docs anytime foreseeable either, sadly.

**Comment by meatballs** [comment]
@hugetim You might be interested in https://github.com/anvilistas/anvil-labs/pull/118
